src/axom/sidre/CMakeLists.txt

```diff
@@ -25,8 +30,7 @@ set(sidre_headers
     core/MapCollection.hpp
     core/SidreTypes.hpp
     core/SidreDataTypeIds.h
-    core/sidre.hpp
-)
+    core/sidre.hpp )
```

Comment:
As a style point, I like having the paren on a line by itself so it's possible to add an additional line without moving the paren.

Reply:
Thanks @ltaylor16 -- I created https://github.com/LLNL/axom/issues/143 for consensus on this.

The last time I spoke to @white238 about this, it seemed like less whitespace was preferable for our build system.